api/types/container/container.go

```diff
@@ -128,3 +129,61 @@ type Summary struct {
 	NetworkSettings *NetworkSettingsSummary
 	Mounts          []MountPoint
 }
+
+// ContainerNode stores information about the node that a container
+// is running on.  It's only used by the Docker Swarm standalone API.
+//
+// Deprecated: ContainerNode was used for the classic Docker Swarm standalone API. It will be removed in the next release.
+type ContainerNode struct {
+	ID        string
+	IPAddress string `json:"IP"`
+	Addr      string
+	Name      string
+	Cpus      int
+	Memory    int64
+	Labels    map[string]string
+}
+
+// Base contains response of Engine API GET "/containers/{name:.*}/json"
+// for API version 1.18 and older.
+//
+// TODO(thaJeztah): combine Base and InspectResponse into a single struct.
+// The split between Base (ContainerJSONBase) and InspecResponse (InspectResponse)
+// was done in commit 6deaa58ba5f051039643cedceee97c8695e2af74 (https://github.com/moby/moby/pull/13675).
+// ContainerJSONBase contained all fields for API < 1.19, and InspectResponse
+// held fields that were added in API 1.19 and up. Given that the minimum
+// supported API version is now 1.24, we no longer use the separate type.
+type Base struct {
```

Comment:
Nit: `InspectBase` would be slightly better IMO. Mainly because it's coupled with `InspectResponse` and we'd like to combine them anyway.

Reply:
Updated 👍